include/launch_core/dev/sensor.h

```diff
@@ -71,5 +71,47 @@ int l_get_sensor_data(const struct device *const dev, int num_channels, enum sen
 int l_get_sensor_data_float(const struct device *const dev, int num_channels, enum sensor_channel const *channels,
                             float **values);
 
+/**
+ * Retrieve data from an accelerometer device and assign converted values to a struct
+ * @param dev - Device to read from
+ * @param p_accel_data - Pointer to an accelerometer data struct
+ * @return Bitfield of failed retrievals
+ */
+int l_get_accelerometer_data_float(const struct device *const dev, l_accelerometer_data_t *p_accel_data);
+
+/**
+ * Retrieve data from an barometer device and assign converted values to a struct
+ * @param dev - Device to read from
+ * @param p_accel_data - Pointer to an accelerometer data struct
```

Comment:
Multiple copy-paste mistakes in the docstrings here.

Reply:
Fixed